cypress/fixtures/analysis.json

```diff
@@ -244,5 +244,26 @@
     "source_analysis_on_pet_clinic_app": {
         "source": "Source code",
         "appName": "pet-clinic"
+    },
+
+    "source+dep_analysis_on_daytrader-app-affected-files": {
+        "source": "Source code + dependencies",
+        "target": ["Application server migration to"],
+        "appName": "deps",
+        "storyPoints": 24,
```

Comment:
We are not validating story points anymore . This can be removed.

Reply:
Done, Thanks for noticing